e2e/fuzz/replay/replay_message.go

```diff
@@ -31,8 +31,8 @@ func NewReplayTimeoutMessage(to pbft.NodeID, msgType pbft.MsgType, view *pbft.Vi
 	}
 }
 
-// ConvertToByteArrays converts ReplayMessage slice to JSON representation and return it back as slice of byte arrays
-func ConvertToByteArrays(messages []*ReplayMessage) ([][]byte, error) {
+// ConvertMessagesToByteArrays converts ReplayMessage slice to JSON representation and return it back as slice of byte arrays
+func ConvertMessagesToByteArrays(messages []*ReplayMessage) ([][]byte, error) {
```

Comment:
This one could be removed if it turns out that generic flavor of fore-mentioned function does its job.